m4/pdns_with_geo.m4

```diff
@@ -1,7 +1,46 @@
 AC_DEFUN([PDNS_CHECK_GEOIP], [
-  PKG_CHECK_MODULES([GEOIP], [geoip],[],
-    AC_MSG_ERROR([Could not find libGeoIP])
+  PKG_CHECK_MODULES([GEOIP], [geoip], AC_DEFINE([HAVE_GEOIP], [1], [Define this if you have GeoIP]), [have_geoip=0])
+  AC_ARG_WITH([maxminddb_incdir],
```

Comment:
libmaxminddb has pkgconfig support: https://github.com/maxmind/libmaxminddb/blob/master/src/libmaxminddb.pc.in
But this is not in the Ubuntu Artful package :(

Reply:
yes, i know. I can add pkgconfig attempt first.

Reply:
artful has it: https://packages.ubuntu.com/artful/amd64/libmaxminddb-dev/filelist

Reply:
in fact, if you look few lines forward, you'll notice that there is PKG_CHECK_MODULES for MMDB

Reply:
@pieterlexis ?

Reply:
:+1: